.github/workflows/ci.yaml

```diff
@@ -129,13 +129,13 @@ jobs:
         with:
           go-version: 1.18
       - name: Setup Minikube
-        uses: manusa/actions-setup-minikube@v2.7.1
+        uses: manusa/actions-setup-minikube@3856c6fa039819f1c8e7e248b1fc5a8564e354c9 # v2.9.0
         with:
-          minikube version: 'v1.21.0'
-          kubernetes version: 'v1.21.0'
+          minikube version: 'v1.31.2'
+          kubernetes version: 'v1.28.1'
```

Comment:
> https://github.com/devfile/api/issues/1295#issuecomment-1771315601

@thepetk Based on your findings, is the Kubernetes version bump necessary or is minikube version bumps sufficient? Though a Kubernetes bump to `v1.28.1` is the plan with upcoming changes it would be better to bump it with the changes for https://github.com/devfile/api/issues/1274 if it does not impact this fix.